Game/Source/GDKTestGyms/Private/NFRConstants.cpp

```diff
@@ -7,24 +7,48 @@
 
 DEFINE_LOG_CATEGORY(LogNFRConstants);
 
-bool UNFRConstants::SamplesForFPSValid() const
+bool UNFRConstants::DoValidityCheck(bool& IsValid, int64 TimeToStart) const
```

Comment:
Nit:
What if we wrap the timing code (and time that it becomes valid) up in a little struct which we can call .IsValid which internally does the caching behavior rather than passing references to a bool here

Reply:
Sure sounds good.